README.md

```diff
@@ -1 +1,89 @@
-# art-institute-chicago-api
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Art Institute Chicago api
+
+The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved on external API.
+
+# Screen Shots
+
+> Comming Soon.
+
```

Comment:
- [ ] **[OPTIONAL]** To make your project more descriptive and professional, I suggest you add a screenshot of your project and also a Live demo link so that users could have a look at your project without having to run it locally to see it.